x-pack/plugins/observability_solution/logs_data_access/server/services/get_logs_rates_service/index.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ElasticsearchClient } from '@kbn/core/server';
+import { estypes } from '@elastic/elasticsearch';
+
+interface Params {
+  esClient: ElasticsearchClient;
+  serviceNames: string[];
+  identifyingMetadata: string;
+  timeFrom: number;
+  timeTo: number;
+}
+
+interface LogErrorsAggregation extends estypes.AggregationsStringRareTermsBucketKeys {
+  logErrors: estypes.AggregationsTermsAggregateBase<estypes.AggregationsStringRareTermsBucketKeys>;
+}
+
+interface LogRateQueryAggregation {
+  services: estypes.AggregationsTermsAggregateBase<LogErrorsAggregation>;
+}
+
+export function createGetLogsRatesService() {
+  return async ({ esClient, identifyingMetadata, serviceNames, timeFrom, timeTo }: Params) => {
+    const esResponse = await esClient.search({
+      index: 'logs-*-*',
```

Comment:
should this be configurable? what about remote clusters?

Reply:
I think we can use `getLogView` from `logsShared` plugin to retrieve the index patterns.

Reply:
We discussed about it being hardcoded for now, @tommyers-elastic is looking into replacements for what is proposed above (`getLogView `).

Reply:
yeh we will add some methods to get the configured logs index patterns shortly